packages/website/utils/sidebarLinks.json

```diff
@@ -290,6 +290,10 @@
     {
       "title": "Getting started",
       "slug": "/getting-started"
+    },
+    {
```

Comment:
Should this link be visible in the sidebar?
<img width="617" alt="image" src="https://user-images.githubusercontent.com/10744462/159278320-fb544319-b888-4e9e-9ee0-a3aecab46984.png">